selenium_tests/lti_emailer/page_objects/add_user_po.py

```diff
@@ -0,0 +1,43 @@
+__author__ = 'hic048'
+
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base import EmailerBasePage
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import Select
+
+# These are page-specific locators.
+class Locator3(object):
```

Comment:
`Locator1`, 2, 3 is pretty opaque -- perhaps we can use the same convention we use in e.g. icommons_lti_tools (e.g. `IndexPageLocators`, `CourseSelectionPageLocators`, etc.)?